src/item.cpp

```diff
@@ -2594,6 +2602,10 @@ std::string item::info( std::vector<iteminfo> &info, const iteminfo_query *parts
                        parts->test( iteminfo_parts::DESCRIPTION_RECHARGE_UPSCAPABLE ) ) {
                 info.push_back( iteminfo( "DESCRIPTION",
                                           _( "* This tool has a <info>rechargeable power cell</info> and can be recharged in any <neutral>UPS-compatible recharging station</neutral>. You could charge it with <info>standard batteries</info>, but unloading it is impossible." ) ) );
+            } else if( has_flag( "RECHARGE_AIR" ) &&
+                       parts->test( iteminfo_parts::DESCRIPTION_RECHARGE_UPSCAPABLE_AIR ) ) {
+                info.push_back( iteminfo( "DESCRIPTION",
+                                          _( "* This tool has a <info>rechargeable air cell</info> and can be recharged in any <neutral>air-compatible recharging station</neutral>. You could charge it with <info>air</info>, but unloading it is impossible." ) ) );
```

Comment:
This copypaste is awful.
```suggestion
                                          _( "* This tool has an <info>air cylinder</info> and can be refilled with air in any <neutral>air pumping station</neutral>. You could fill it with <info>air</info>, but unloading it is impossible." ) ) );
```